tests/console/php7_postgresql96.pm

```diff
@@ -30,7 +30,7 @@ sub run {
     setup_apache2(mode => 'PHP7');
 
     # install requirements
-    zypper_call 'in php7-pgsql postgresql96-server sudo';
+    zypper_call 'in php7-pgsql postgresql*-server sudo';
```

Comment:
That does not make any sense. The test module is called php7_postgresql96 and you install any version now. There is also a package "postgresql-server" which should be used instead.

You could rename the test module but there is already tests/console/postgresql_server.pm so please sort out this mess overall, also including tests/console/php5_postgresql96

Reply:
Hi Oliver
I have done some research before modified this code.
1. SLES12SPx only have postgresql96-server and SLES15 only have postgresql10-server. It seems there is only version for each sles.
2. I have try parameter both postgresql*-server and postgresql-server10( postgresql96-server) with zypper in. The lists of package to be installed are same. So I think this command should be work as expect.
3. I can't find any package on SLES12SP3 by "zypper se postgresql-server". That's why I use postgresql*-server instead.

I agree the name php7_postgresql96 is not suitable anymore, it is a liitle mess here.
Here is two suggestion which could be helpful, 
1.  Add  `sle_version_at_least('15') ? 'postgresql10-server' : 'postgresql96-server'` before install postgresql. (same as tests/console/postgresql_server.pm)
2. wrap Up postgresql install and add it to `lib/apachetest.pm`
Which one do you think is better? And please feel to add  any suggestion. Thanks 

And one more question about it, what is the different between php7_postgresql96 and postgresql_server.pm?

Reply:
> And one more question about it, what is the different between php7_postgresql96 and postgresql_server.pm?

Let me start to answer that easy question first :) postgresql_server is only testing postgres itself, php*_postgres* is testing apache in combination with the php plus postgres stack

Thank you for your checking older versions. I was not aware that older SLE versions only have the version specific subpackages of postgres.

postgresql*-server would install *all* available packages and this is most likely not what we want. For openSUSE we have multiple parallel versions so * seems to be a bad idea.

IMHO we should not have versioned files unless we need to. Better be explicit for the package version for SLE12 and use the generic package postgresql-server for all more recent versions